dist/compiler/passes/generateJS.js

```diff
@@ -302,7 +302,16 @@ var generateJavascript = function generateJavascript(ast, options) {
       }
     }
   };
-  return codeGenerator(options.results.instructions, options.results.code);
+
+  for (var s in htmlSymbols) {
+    options.results.instructions.symbolsMap[s] = htmlSymbols[s];
+  }
+  for (var s in tornadoLeafSymbols) {
+    options.results.instructions.symbolsMap[s] = tornadoLeafSymbols[s];
+  }
+  for (var s in tornadoBodySymbols) {
+    options.results.instructions.symbolsMap[s] = tornadoBodySymbols[s];
+  }
```

Comment:
We are currently only modifying the symbolsMap. Next step would be to modify the instructions list if